src/features/threads/DisplayTempUb3rStatus.js

```diff
@@ -0,0 +1,27 @@
+const Feature = require("../../core/Feature");
+const Threads = require("../../sections/Threads");
+
+class DisplayTempUb3rStatus extends Feature {
+  constructor() {
+    super({
+      section: Threads,
+      name: "Display Temp Ub3r Status",
+      default: true,
+      description: "Displays the temp Ub3r status in the postbit for users who do not have permanent Ub3r.",
```

Comment:
`Displays the temp Ub3r status in the postbit for users who do not have permanent Ub3r.`
Incase users do not know what postbit is:
`Displays the expiration date below awards for Ub3rs who do not have permanent Ub3r.`

Reply:
modified description